server/gc/src/test/java/org/apache/accumulo/gc/GarbageCollectionTest.java

```diff
@@ -778,4 +838,152 @@ public void bulkImportReplicationRecordsPreventDeletion() throws Exception {
     assertEquals(1, gce.deletes.size());
     assertEquals("hdfs://foo.com:6000/accumulo/tables/2/t-00002/A000002.rf", gce.deletes.get(0));
   }
+
+  @Test
+  public void testMissingTableIds() throws Exception {
```

Comment:
```suggestion
  public void testMissingTableIds() {
```

Reply:
Addressed in c4e0ab918bf002c83d4200af67df0578aff44e73